examples/simulated/plot_classifier_comparison.py

```diff
@@ -0,0 +1,216 @@
+
+A comparison of several Riemannian classifiers on low-dimensional synthetic
+datasets, adapted to SPD matrices from [1]_.
+The point of this example is to illustrate the nature of decision boundaries
+of different classifiers.
+This should be taken with a grain of salt, as the intuition conveyed by
+these examples does not necessarily carry over to real datasets.
+
+The plots show training points in solid colors and testing points
+semi-transparent. The lower right shows the classification accuracy on the test
+set.
+
+"""
+# Authors: Quentin Barthélemy
+#
+# License: BSD (3-clause)
+
+from functools import partial
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import ListedColormap
+from sklearn.model_selection import train_test_split
+
+from pyriemann.datasets import make_covariances, make_gaussian_blobs
+from pyriemann.classification import MDM, KNearestNeighbor
+
+
+###############################################################################
+
+
+@partial(np.vectorize, excluded=['clf'])
+def get_proba(cov_00, cov_01, cov_11, clf):
+    cov = np.array([[cov_00, cov_01], [cov_01, cov_11]])
+    with np.testing.suppress_warnings() as sup:
+        sup.filter(RuntimeWarning)
+        return clf.predict_proba(cov[np.newaxis, ...])[0, 1]
+
+
+###############################################################################
+
+names = [
+    "MDM",
+    "Nearest Neighbors"
+]
+classifiers = [
+    MDM(),
+    KNearestNeighbor(n_neighbors=3)
+]
+n_classifs = len(classifiers)
+
+rs = np.random.RandomState(2022)
+n_matrices, n_channels = 50, 2
+y = np.concatenate([np.zeros(n_matrices), np.ones(n_matrices)])
+
+datasets = [
+    (
+        np.concatenate([
+            make_covariances(
+                n_matrices, n_channels, rs, evals_mean=10, evals_std=1
+            ),
+            make_covariances(
+                n_matrices, n_channels, rs, evals_mean=15, evals_std=1
+            )
+        ]),
+        y
+    ),
+    (
+        np.concatenate([
+            make_covariances(
+                n_matrices, n_channels, rs, evals_mean=10, evals_std=2
+            ),
+            make_covariances(
+                n_matrices, n_channels, rs, evals_mean=12, evals_std=2
+            )
+        ]),
+        y
+    ),
+    make_gaussian_blobs(
+        2*n_matrices, n_channels, random_state=rs, class_sep=3., class_disp=0.5
+    ),
+    make_gaussian_blobs(
+        2*n_matrices, n_channels, random_state=rs, class_sep=1., class_disp=0.7
+    )
+]
+n_datasets = len(datasets)
+
+cm = plt.cm.RdBu
+cm_bright = ListedColormap(["#FF0000", "#0000FF"])
+
+
+###############################################################################
+
+figure = plt.figure(figsize=(8, 10))
+i = 1
+# iterate over datasets
+for ds_cnt, (X, y) in enumerate(datasets):
+    # split dataset into training and test part
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.4, random_state=42
```

Comment:
```suggestion
        X, y, test_size=0.4, random_state=rs
```
Does it make the figure reproducible?